projects/testing-library/src/lib/testing-library.ts

```diff
@@ -73,6 +75,7 @@ export async function render<SutType, WrapperType = SutType>(
     const idAttribute = fixture.nativeElement.getAttribute('id');
     if (idAttribute && idAttribute.startsWith('root')) {
       fixture.nativeElement.removeAttribute('id');
+      cleanup = () => fixture.nativeElement.setAttribute('id', idAttribute);
```

Comment:
I restore only the `id`, because it's the only attribute that Angular need to remove old roots. Should do we restore `ng-version` too?

Reply:
Sorry I wasn't clear, I will follow-up with another PR